operations/mimir/autoscaling.libsonnet

```
@@ -53,6 +53,11 @@
   local replicasWithWeight(replicas, weight) =
     if weight <= 0.5 then std.ceil(replicas * weight) else std.floor(replicas * weight),
 
+  // getScaleDownPeriod will return the scale down period expressed in seconds.
```

[Nit] I think "returns" is more standard verbiage for function documentation, and more explicit for the reader:
```suggestion
  // getScaleDownPeriod returns the scale down period expressed in seconds.
```